src/scene/sceneManager.cpp

```diff
@@ -71,10 +59,14 @@ void SceneManager::setSelectedSceneObject(const SceneObject *sceneObject) {
   }
 }
 
-void SceneManager::addSelectedSceneObject(const SceneObject *sceneObject) {
+void SceneManager::clickSelectionSceneObject(const SceneObject *sceneObject) {
   auto it = std::find_if(this->sceneObjects.begin(), this->sceneObjects.end(), [&](auto &&obj) { return obj.get() == sceneObject; });
   if (it != this->sceneObjects.end()) {
-    this->selectedSceneObjects.push_back(it->get());
+    if (std::find(this->selectedSceneObjects.begin(), this->selectedSceneObjects.end(), it->get()) != this->selectedSceneObjects.end()) {
+      this->selectedSceneObjects.erase(std::remove(this->selectedSceneObjects.begin(), this->selectedSceneObjects.end(), it->get()), this->selectedSceneObjects.end());
```

Comment:
le erase ici, c'est tu l'equivalent d'un this->selectedSceneObjects.clear() ?

Reply:
Yes 
![image](https://github.com/NathPaquet/Infographie_IFT-3100/assets/73801331/f541c3eb-fa30-4447-a7a8-08494d740db0)
![image](https://github.com/NathPaquet/Infographie_IFT-3100/assets/73801331/d392c1b0-8188-462e-b5f6-32c8e0dae53b)

Reply:
tu peux faire un clear alors :)